bundles/ranvier-combat/lib/Combat.js

```diff
@@ -91,18 +92,26 @@ class Combat {
    * @param {Character} target
    */
   static makeAttack(attacker, target) {
-    const amount = this.calculateWeaponDamage(attacker);
+    //defines variables associated with critical strikes based on a random percentage chance.
+    const critChance = Math.max(attacker.getMaxAttribute('critical') || 0, 0);
```

Comment:
Pulling each of these calculations into its own constant makes this way more readable than it would be otherwise, nicely done.

Reply:
I'm not sure if you're asking me to change something or are just complimenting how I did this.

Reply:
I thought about suggesting pulling all crit-related code into a function like `this.calculateCritical(attacker, target, amount)` but overall just thought it was well done.